server/users.js

```diff
@@ -37,6 +37,17 @@ module.exports = require('express').Router()
       .then(user => res.json(user))
       .catch(next)
     })
+  // MPM adding a new route below to get all orders by a user
+  .get('/:id/orders', /* mustBeLoggedIn, */
```

Comment:
This should just be a query on the usersById. If we always want orders with the user this will make more sense